### DISCUSSION TITLE
[x][manuscript][closed] Multiple studies per experiment?

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#100] [Team Posts]
An EGA Dataset (and therefore its experiments) can map to multiple EGA studies... do we wish to reproduce this?

**Comment by sveinugu** [comment]
So I tried to understand the EGA data model:

https://ega-archive.org/submission/sequence/programmatic_submissions
https://ega-archive.org/submission/programmatic_submissions/common-aspects

It is based upon the SRA data model, which is a bit simpler:

https://www.ncbi.nlm.nih.gov/sra/docs/submitmeta/

The way I understand it is that Study, Experiment and Sample (which is in both places) are more or less similar to our concepts, expect that they by experiment mean lab experiment, while in our model this should also include analyses (which is why we have added the derived_by link). A run is the object that points to the data, meaning mostly raw reads (FASTQ...). In EGA they have added the concept of Analyses, which is "analysis data files", which in our case would be the track concept. Analyses and runs are on the same level, as I understand it, the difference being that Runs are sequencing (or array) data, while Analyses are the results of some analyses (e.g. BAM files from mapping).  

A dataset is then simply a collection of Runs and Analyses, i.e. a set of data files. The main need for this would be to collect many related files into a larger collection to simplify storage and access. We do not have anything resembling that concept. I would argue that one of the main reasons for this IS is really that we want to provide easy access to track data at a file-level granularity. Thus, the concept of dataset seems not very relevant for us. As an EGA dataset is a collection of data files (Runs and Analyses), it follows that they can come from different studies. But, as argued above, I don't find that part of the model very relevant for us.

Please correct me if I have misunderstood something. I have just spent a short hour on trying to wrap my head around these things and have no prior experience in either EGA of SRA data models.

**Comment by sveinugu** [comment]
If one of our users want to get access to EGA data, it seems to me that the correct link would be to follow the experiment:global_id, and then locate the corresponding runs (perhaps by using the sample:global_id, as it seems several runs can in some cases share the same experiment: https://ega-archive.org/submission/programmatic_submissions/common-aspects). From the run, one should be able to find the dataset to download. Of course we could provide a direct link to the dataset, probably from the track object, to simplify this. 

I do not know how such a model would map to other external models, though..

**Comment by dzerbino** [comment]
Thinking about it with more perspective, I realise that we discussed this earlier in the document on multiplicity, where we noted "We can assume most tracks are associated to a single study". In effect, the track hub describes a study, but another track hub could describe the same tracks under another study.

**Comment by jmfernandez** [comment]
In the latest ICGC releases, in CLL analysis metadata we had to include the (partial) chain of EGA identifiers, _EGA Study:EGA Dataset:EGA File_ , so the study, dataset and specific file containing the data linked to that metadata could be traced

**Comment by sveinugu** [comment]
@jmfernandez I cannot find the concept of EGA File in the model (URL above). In my understanding a Run and Analysis are both files. Or am I missing something?

**Comment by sveinugu** [comment]
@jmfernandez So according to the examples here:

https://www.ebi.ac.uk/ega/submission/sequence/programmatic_submissions/prepare_xmls#dataset_XML

It seems a Dataset points to Runs and Analyses, while these again point to files. It does not change my argument, that the dataset concept seems unnecessary for us. However, a link to an EGAF accession from a track object seems a nice thing. This would then actually be a global_id of some sort, but only a global id of the data file. On the other hand, I don't know if EGAFs can be located without the dataset and/or run/analysis accessions. 

Regardless of the complexities above, I think we should check how feasible it is to locate the correct EGA file from a track as it is now, and if that is hard, we should look at improving the metadata.

**Comment by jmfernandez** [comment]
@sveinugu , AFAIK right now it is not possible (at least, from the [new EGA web interface](https://ega-archive.org)) to search by EGA file identifier. Next page describes the different kind of identifiers managed by EGA:

https://ega-archive.org/submission/programmatic_submissions/common-aspects

AFAIK, simplifying some concepts, a DACO controls its studies, datasets and files (among other concepts). Studies can be based on more than one dataset, using some or all the files (and their associated metadata) from those datasets, and datasets can be (partially) reused in several studies. A dataset is composed by one or more files, for instance https://ega-archive.org/datasets/EGAD00010000875/files

**Comment by dzerbino** [comment]
I think this conversation is drifting a bit. The files of interest in the trackhub are BigWigs and BigBeds, which are generally not in EGA (if they were, we'd just get the metadata from EGA and would not bother with trackhubs in the first place!)

**Comment by sveinugu** [comment]
The point being that we should be able to locate the raw data file from the metadata, for example if one wants to redo the peak calling. As we are using BLUEPRINT as our initial track hub, I think we should make sure that the metadata is as FAIR as possible, within reason. But agree that we (meaning I) have drifted a bit.

**Comment by sveinugu** [comment]
So 3 conclusions to this point, as I understand them:
  - We ignore the EGA Dataset concept (which was the starting point)
  - We add a raw file url attribute for tracks. There is an issue on this: https://github.com/fairtracks/fairtracks_standard/issues/6
  - BLUEPRINT will have nothing here, unfortunately

**Comment by sveinugu** [comment]
As there is an issue, I am closing this, but keeping it pinned for the manuscript discussion.
